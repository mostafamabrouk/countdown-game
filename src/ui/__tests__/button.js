/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import { Button } from "../";
import "jest-styled-components";

test("Button renders the same", () => {
  const component = renderer.create(<Button>Label</Button>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
