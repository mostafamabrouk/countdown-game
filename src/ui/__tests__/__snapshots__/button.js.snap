// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders the same 1`] = `
.c0 {
  display: block;
  padding: 15px;
  border: none;
  background: #009680;
  color: #fff;
  margin: 20px auto;
  font-size: 16px;
  text-transform: uppercase;
  font-weight: 200;
  -webkit-letter-spacing: 3px;
  -moz-letter-spacing: 3px;
  -ms-letter-spacing: 3px;
  letter-spacing: 3px;
  cursor: pointer;
  -webkit-transition: 0.2s all ease;
  transition: 0.2s all ease;
}

.c0:hover {
  background: #000;
}

<button
  className="c0"
>
  Label
</button>
`;
